lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -229,24 +229,24 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
         #     # of mongo queries,
         #     # of xblocks
         # )
-        ('no_overrides', 1, True, False): (47, 1, 6, 13),
-        ('no_overrides', 2, True, False): (119, 16, 6, 84),
-        ('no_overrides', 3, True, False): (399, 81, 6, 335),
-        ('ccx', 1, True, False): (47, 1, 6, 13),
-        ('ccx', 2, True, False): (119, 16, 6, 84),
-        ('ccx', 3, True, False): (399, 81, 6, 335),
-        ('ccx', 1, True, True): (47, 1, 6, 13),
-        ('ccx', 2, True, True): (119, 16, 6, 84),
-        ('ccx', 3, True, True): (399, 81, 6, 335),
-        ('no_overrides', 1, False, False): (47, 1, 6, 13),
-        ('no_overrides', 2, False, False): (119, 16, 6, 84),
-        ('no_overrides', 3, False, False): (399, 81, 6, 335),
-        ('ccx', 1, False, False): (47, 1, 6, 13),
-        ('ccx', 2, False, False): (119, 16, 6, 84),
-        ('ccx', 3, False, False): (399, 81, 6, 335),
-        ('ccx', 1, False, True): (47, 1, 6, 13),
-        ('ccx', 2, False, True): (119, 16, 6, 84),
-        ('ccx', 3, False, True): (399, 81, 6, 335),
+        ('no_overrides', 1, True, False): (48, 1, 6, 13),
+        ('no_overrides', 2, True, False): (120, 16, 6, 84),
```

Comment:
@adampalay on progress page I added a query to check the user enrollment that query increase the count.